Engine/DataFeeds/TimeSlice.cs

```diff
@@ -293,7 +312,7 @@ private static bool HandleOptionData(DateTime algorithmTime, BaseData baseData,
             var symbol = baseData.Symbol;
             
             OptionChain chain;
-            var canonical = Symbol.Create(symbol.ID.Symbol, SecurityType.Option, symbol.ID.Market);
+            var canonical = Symbol.CreateOption(symbol.Underlying, symbol.ID.Market, default(OptionStyle), default(OptionRight), 0, SecurityIdentifier.DefaultDate);
```

Comment:
Good question. Like I mentioned I plan those changes later to harmonize symbols.